<launch>
    <node pkg="vision_cp" type="visionCP" name="visionCP">
	</node>
</launch>
